tests/conftest.py

```
@@ -1,7 +1,55 @@
+    Base.metadata.create_all(bind=engine)
+    yield engine
+    # Drop all tables after tests
+    Base.metadata.drop_all(bind=engine)
+
+@pytest.fixture
+def db_session(db_engine) -> Generator:
+    """Create a test database session."""
+    connection = db_engine.connect()
+    transaction = connection.begin()
+    session = SessionLocal(bind=connection)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)